output_unifier.py

```diff
@@ -248,6 +346,29 @@ def _standardizer_tokei(data):
     return formatted_output
 
 
+def _standardizer_rust_code_analysis(data):
+    formatted_output = {"files": []}
+
+    metrics = data["metrics"]
+
+    logging.debug(metrics)
+
+    per_file = {
+        "filename": data["name"],
+        "LOC": metrics["sloc"],
+        "CLOC": int(metrics["sloc"]) - int(metrics["lloc"]),
+        "CYCLOMATIC": metrics["cyclomatic"],
+        "HALSTEAD": metrics["halstead"],
+        "NARGS": metrics["nargs"],
+        "NEXITS": metrics["nexits"],
+        "functions": metrics["spaces"],
```

Comment:
Does `metric["spaces"]` follow our "standard"?

Are this `per_file` keys equal to the already used ones? Are they different for a reason? (maybe testing purposes)

Reply:
Which is the standard? Is the one contained in the `Definition.json` file? If it is, should I use the version 2 of the standard?

For what concerns the `per_file`, I've added more metrics. Should I move them elsewhere?

Reply:
Yes, the one defined here: `Definition.json`.
Use the Version 1, the one _with_ the Global Stats. The 2nd one can be removed

The `Definition.json` file is like an example file, used some weeks ago as a guide while writing the Aggregator from scratch. There is no real need to keep it up-to-date, it is there only for convenience.

Reply:
I see, thanks a lot for your explanation! 
I will change the PR according to the `Definition.json` example. 

About the `per_file` object, can I add there the additional metrics?

Reply:
Yes, you can. If those metrics analyse a single file, you have to put them there